clientv3/client.go

```diff
@@ -97,7 +97,12 @@ func (c *Client) Close() error {
 func (c *Client) Ctx() context.Context { return c.ctx }
 
 // Endpoints lists the registered endpoints for the client.
-func (c *Client) Endpoints() []string { return c.cfg.Endpoints }
+// It returns the copy of endpoints to prevent accidental endpoint updates.
```

Comment:
remove the comment from func description. move that into func itself. it is an implementation detail.